Model/MediaInterface.php

```diff
@@ -365,4 +365,16 @@ public function getGalleryHasMedias();
      * @return string
      */
     public function getPreviousProviderReference();
+
+    /**
+     * @return array
+     */
+    public function getAssetPaths();
```

Comment:
please remove this and create a new PR where we add these methods to the interface, otherwise its a BC break